.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build ARM user
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install deps
+      run: |
+        apt --no-install-recommends -y build-dep qemu
+        apt install autoconf libtool protobuf-c-compiler
+        opam install piqi
```

Comment:
There's no opam in ubuntu:latest, you have to install it. Use [setup-ocaml][1] action for that, e.g., like [this][2]
```
      - name: Install OCaml
        uses: ocaml/setup-ocaml@v2
        with:
          ocaml-compiler: 4.14.x
          dune-cache: true
          opam-disable-sandboxing: true
```


[1]: https://github.com/ocaml/setup-ocaml
[2]: https://github.com/BinaryAnalysisPlatform/bap/blob/master/.github/workflows/full.yml#L28